src/tests/lgr_webclient_test_base.py

```diff
@@ -0,0 +1,20 @@
+from django.test import TestCase
+
+from lgr_auth.models import LgrUser
+
+
+class LgrWebClientTestBase(TestCase):
```

Comment:
you are sharing a piece of code between apps. We try to avoid that now as we may need to actually split the apps in multiple django projects. If you have one project that may be a dependency for the others, put that in here if this makes sense